src/main/java/Wolox/training/controllers/BookController.java

```diff
@@ -19,5 +26,36 @@ public String home() {
         return "home";
     }
 
+    // Create
+    @PostMapping
+    @ResponseStatus(HttpStatus.CREATED)
+    public Book create(@RequestBody Book book) {
+        return (Book)bookRepository.save(book);
+    }
+
+    // Read
+    @GetMapping
+    public Iterable findAll() {
+        return bookRepository.findAll();
+    }
+
+    @GetMapping("/{id}")
+    public Optional findById(@PathVariable int id) {
+        return bookRepository.findById(id);
+    }
+
+    // Update
+    @PutMapping("/{id}")
```

Comment:
in this method you are not updating the model. 
You fund the book using the id but then saving the same one. 
Check the documentations, you hace to receive a book as a parameter.